CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Feature Enhancements:
   Implement global `Infinity` property (@AnirudhKonduru)
 - [FEATURE #410](https://github.com/boa-dev/boa/pull/410):
   Add support for the reviver function to JSON.parse (@abhijeetbhagat)
+- [FEATURE #525](https://github.com/boa-dev/boa/pull/425):
+  Implement additional `Math` methods (@mr-rodgers)
```

Comment:
why did you change this to @mr-rodgers?
I don't see him on that PR

Reply:
Sorry, when I copy/pasted the [FEATURE #525], I forgot to update the link.

Didn't get the #514 part though, it is mentioned below in [Internal #514].

Reply:
#525 looks like its still open so shouldn't really be on here.
`0.9.0` has been closed now, so any remaining PRs will go into `0.10.0`